TheJumpCloud/support
# CUT-4490-ImportUpdate CSV Bug Fix

## Issues
* [CUT-4490](https://jumpcloud.atlassian.net/browse/CUT-4490) - CUT-4490-CSVImportUpdate-Sort-Bug

## What does this solve?
Fixes an issue when importing/updatingCSV user/s with more than 10 custom attributes
## Is there anything particularly tricky?
n/a
## How should this be tested?
Import-JCUsersFromCSV
1. Import a user with 10 or more custom attributes
[NewUserCSV.csv](https://github.com/user-attachments/files/18013569/NewUserCSV.csv)
2. User creation should not return an error and create the user and custom attributes
![image](https://github.com/user-attachments/assets/af701b77-6314-4e09-ab07-2c7ace6254b9)
<img width="2047" alt="image" src="https://github.com/user-attachments/assets/ea997bd6-1688-4d89-a2fe-de2c0c5258f6">


Update-JCUsersFromCSV
1. Update a user with 10 or more custom attributes
[UpdateUser.csv](https://github.com/user-attachments/files/18013621/UpdateUser.csv)
4. User update should not return an error and update the custom attributes
<img width="1733" alt="image" src="https://github.com/user-attachments/assets/ccd40ae2-a73a-45d9-8d60-c757d732f88c">

## Screenshots
